packit/api.py

```diff
@@ -233,6 +234,10 @@ def _get_sandcastle_exec_dir(self):
 
         return SANDCASTLE_EXEC_DIR
 
+    @property
+    def pkg_tool(self) -> str:
+        return self.package_config.pkg_tool or self.config.pkg_tool
```

Comment:
```suggestion
    @property
    def pkg_tool(self) -> str:
        """Returns the packaging tool. Prefers the package-level override."""
        return self.package_config.pkg_tool or self.config.pkg_tool
```

I would put the same comment here as well.